thinkhazard/static/less/report.less

```diff
@@ -102,6 +102,23 @@ body {
   }
 }
 
+.hazard-title-level(@level) {
+  @color: "level-@{level}-color";
+  .level-@{level} {
+    .page-header {
+      border-color: @@color;
+    }
+    .level {
+      color: @@color;
+    }
```

Comment:
This has an unexpected effect: the legend title is colored now. It shouldn't be.